client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -775,6 +775,17 @@ Undocumented.prototype.getTitanOrderProvisioningURL = function ( domain, fn ) {
 	);
 };
 
+Undocumented.prototype.getTitanIncomingURL = function ( mode, jwt, fn ) {
```

Comment:
Nit: I can't tell what this does from the function name. (Though I also realise it's hard to describe what is going on here!)
Maybe something like `getTitanDetailsForIncomingRedirect`?
(Not blocking, but noting it early!)